src/components/NewAppModal.js

```diff
@@ -144,6 +144,10 @@ export const NewAppModal = _.flow(
                 ) + persistentDiskCost({ size: 30, status: 'Running' })
               )} per hr`)
             ])
+          ]),
+          h(Link, { href: 'https://support.terra.bio/hc/en-us/articles/360050566271', ...Utils.newTabLinkProps }, [
```

Comment:
This link doesn't seem to be published yet, it prompted me to log in. That may be expected at this point, though.

Reply:
Yeah, Anton said not to worry about that. https://broadinstitute.slack.com/archives/C33EBDEAX/p1601923150091100